packages/artifacts/CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Aragon OSx artifacts
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## v1.4
+
+### Changed
+
+- Shipping the ABI and addresses for OSx v1.4
+- Exporting a lighter package
```

Comment:
if we don't remove bytecode and also it will be backward compatible, how it is lighter?

Reply:
Fair. Still, we are exporting only 2 fields, whereas typechain used to export a lot more. Also, not all contracts are being exported, only the relevant ones.

Happy to edit these lines if you so prefer